src/encoded/authentication.py

```diff
@@ -279,19 +340,41 @@ def logout(context, request):
     The front-end handles logging out by discarding the locally-held JWT from
     browser cookies and re-requesting the current 4DN URL.
     """
-    #request.session.invalidate()
-    #request.response.headerlist.extend(forget(request))
+    request_parts = urlparse(request.referrer)
```

Comment:
Does request really spell `referrer` correctly? I thought it was lost in antiquity that we were doomed to see "referer" spelled wrong in all header situations. As long as the right thing is happening here, I'm fine. But my eyes are now trained to expect a misspelling.

Reply:
Haha.. at least in [recent version it looks like it has both](https://gyazo.com/1f77af171b1bff3237af0343d757e82b).